pandas/tests/plotting/common.py

```diff
@@ -13,6 +13,7 @@
 from pandas import DataFrame, Series
 import pandas._testing as tm
 
+
 """
 This is a common base class used for various plotting tests
 """
```

Comment:
does this docstring belong inside the following class

Reply:
yes, updated